resippy/photogrammetry/nav/abstract_nav.py

```diff
@@ -0,0 +1,33 @@
+import abc
+from six import add_metaclass
+
+
+@add_metaclass(abc.ABCMeta)
+class AbstractNav:
+
+    def __init__(self):
+        self._num_records = 0
+        self._record_length = 0
+        self._nav_data = None
+
+    @abc.abstractmethod
+    def get_nav_record(self,
```

Comment:
Overall I like the way this is set up.  Should we add more functionality though?  For instance, maybe keep get_nav_record, but then make other methods like "get_roll", "get_pitch", "get_heading", "get_lat", "get_lon", "get_alt" that just use the nav_record?  Trying to think through all of the things a user might need here.  The main thing I'm thinking is that we can't tell what the dictionary keys are, or what the useable records exist, so without that knowledge a user can't directly access records without hitting a breakpoint in the code and checking the dictionary entries from get_nav_record. 

Could also potentially add in interpolation options.  Also, maybe add a method to get CRS data and datum references?  A lot of these can be placeholders for now.